CIV/Funds/CIV.rdf

```diff
@@ -1133,7 +1129,7 @@
 		<rdfs:subClassOf rdf:resource="&fibo-fbc-pas-caa;Account"/>
 		<rdfs:subClassOf>
 			<owl:Restriction>
-				<owl:onProperty rdf:resource="&fibo-fnd-accx-acc;accountHasProvider"/>
+				<owl:onProperty rdf:resource="&fibo-fnd-pty-pty;hasPartyInRole"/>
```

Comment:
I don't like the restriction on FundsProcessingAccount on hasPartyInRole which is too general as to what the role actually is and not an adequate replacement for the original hasProvider.

Reply:
I've revised this to use isProvidedBy from Relations - I had changed it originally to follow the pattern we have in Account, but I've also revised the patterns on the Account class to use isProvidedBy as appropriate.